src/Init/WF.lean

```diff
@@ -31,6 +42,13 @@ def inv {x y : α} (h₁ : Acc r x) (h₂ : r y x) : Acc r y :=
 
 end Acc
 
+/--
+A relation `r` is `WellFounded` if all elements of `α` are accessible within `r`.
+If a relation is `WellFounded`, it does not allow for an infinite descent along the relation.
+
+If the arguments of the recursive calls in a function definition decrease according to
+a well founded relation, then the function terminates.
+-/
```

Comment:
```suggestion

A well-founded relation is also called Artinian, and has the “descending chain condition”.
-/
```

Reply:
Which users would be helped by this addition? It seems to me to be a side remark, but I also don't know the whole audience intimately.

Reply:
It's helpful to use names from the literature in docstrings so that if someone searches for "descending chain condition" they will find this definition. There is no need to draw attention to these words, they are basically just keywords for SEO.

Reply:
There's always a balance between the SEO aspect and the sheer intimidation of having lots of big words that few people understand. I'm not sure where that balance is here, for half our target audience; I'm inclined to trust you if you think the benefits outweigh the costs in this case.

Reply:
I've got nothing against highfalutin' book-talk when it's helpful - but there is a kind of self-parody effect that it'd be good to avoid. Like I said - I'm inclined to trust you if you think this balance works out here for the mathematician audience.

If we do want:

> additional wording around it to make it clear that if you don't know what it's talking about you can safely ignore the remark

then we could say

```suggestion
Well-founded relations are sometimes called _Artinian_ or said to satisfy the “descending chain condition”.
-/
```